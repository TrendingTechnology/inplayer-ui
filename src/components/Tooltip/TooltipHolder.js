import styled from 'styled-components';

const TooltipHolder = styled.div`
  position: relative;
`;

export default TooltipHolder;
