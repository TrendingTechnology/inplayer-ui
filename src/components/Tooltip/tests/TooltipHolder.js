import { generateSnapshots } from 'helpers';

import TooltipHolder from '../TooltipHolder';

generateSnapshots(TooltipHolder);
