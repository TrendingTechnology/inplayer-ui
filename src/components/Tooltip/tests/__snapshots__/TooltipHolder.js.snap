// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TooltipHolder renders correctly 1`] = `
.c0 {
  position: relative;
}

<div
  className="c0"
/>
`;
